src/streams.c

```diff
@@ -427,7 +427,7 @@ static void READ_TEST_OR_LOOP(Obj context)
 Int READ_GAP_ROOT ( const Char * filename )
 {
     TypGRF_Data         result;
-    Int                 res;
+    int                 res;
```

Comment:
What his the reason for changing this?

Reply:
That's to avoid a compiler warning in the `Panic` call, where `%d` is used which should indicate an `int` argument; on most 64bit systems, `Int` is a bigger type. Anyway, this part of the PR actually conflicts with PR #3818, so I'll remove this here.